File: docs/components_page/components/layout/horizontal.R
library(dashBootstrapComponents)

row_content <- list(
  dbcCol(div("One of two columns"), width = 4),
  dbcCol(div("One of two columns"), width = 4)
)

row <- div(
  list(
    dbcRow(
      row_content,
      justify = "start"
    ),
    dbcRow(
      row_content,
      justify = "center"
    ),
    dbcRow(
      row_content,
      justify = "end"
    ),
    dbcRow(
      row_content,
      justify = "between"
    ),
    dbcRow(
      row_content,
      justify = "around"
    ),
    dbcRow(
      row_content,
      justify = "evenly"
    )
  )
)

File: docs/components_page/components/layout/horizontal.jl
using DashBootstrapComponents

row_content = [
    dbc_col(html_div("One of two columns"), width = 4),
    dbc_col(html_div("One of two columns"), width = 4),
];

row = html_div([
    dbc_row(row_content, justify = "start"),
    dbc_row(row_content, justify = "center"),
    dbc_row(row_content, justify = "end"),
    dbc_row(row_content, justify = "between"),
    dbc_row(row_content, justify = "around"),
    dbc_row(row_content, justify = "evenly"),
]);
